src/chrome/content/rules/Suddeutsche_Zeitung.xml

```diff
@@ -144,11 +123,49 @@
 
 	<securecookie host="^(?:anzeigen-buchen|epaper|id|service|stellenmarkt|sz-shop|weiterbildung)\.sueddeutsche\.de$" name=".+" />
 
-
 	<rule from="^http://polpix\.sueddeutsche\.com/"
 		to="https://www.sueddeutsche.de/" />
 
-	<rule from="^http://(anzeigen(?:-buchen)?|epaper|geoservice|id|maerkte|(?:immobilien|motor|stellen)markt|maps|service|sz-shop|szshop|weiterbildung|www)\.sueddeutsche\.de/"
+	<rule from="^http://(anzeigen(?:-buchen)?|epaper|geoservice|global|id|jetzt|maerkte|(motor|stellen)markt|service|sz-shop|weiterbildung)\.sueddeutsche\.de/"
 		to="https://$1.sueddeutsche.de/" />
 
+	<rule from="^http://immobilienmarkt\.sueddeutsche\.de/(cms/data/|cms/fileadmin/|cms/uploads/|css/|favicon\.ico|gfx/)"
+		to="https://immobilienmarkt.sueddeutsche.de/$1" />
+
+	<rule from="^http://maps\.sueddeutsche\.de/css/"
+		to="https://maps.sueddeutsche.de/$1" /> <!--subdomain can be secured completely but creates false MCB -->
+
+	<rule from="^http://szshop\.sueddeutsche\.de/(favicon\.ico|out/sz/src/)"
+		to="https://szshop.sueddeutsche.de/$1" /> <!--subdomain can be secured completely but creates false MCB -->
+
+	<rule from="^http://www\.sueddeutsche\.de/favicon\.ico"
+		to="https://www.sueddeutsche.de/$1" />
```

Comment:
https://www.sueddeutsche.de/favicon.ico redirects to http://www.sueddeutsche.de/favicon.ico

There is no () group in the from for $1.